pro-planner-client/src/components/ErrorToast.jsx

```diff
@@ -27,36 +27,38 @@ const ErrorToast = forwardRef((props, ref) => {
     let errIcon;
     switch (errType) {
         case ERR_TYPE.ERR:
-            errIcon = <MdDangerous className="error-icon" color="red" />
+            errIcon = <MdDangerous className="error-icon" color="red"/>
             break;
         case ERR_TYPE.WARN:
-            errIcon = <IoWarning className="error-icon" color="orange" />
+            errIcon = <IoWarning className="error-icon" color="orange"/>
             break;
         case ERR_TYPE.INFO:
-            errIcon = <BsInfoCircleFill className="error-icon" color="navy" />
+            errIcon = <BsInfoCircleFill className="error-icon" color="navy"/>
             break;
         default:
 
     }
 
     return <>
-        {showError 
-        && <Toast
-            className="m-3 position-relative err-toast"
-        >
-            <Toast.Header className="d-flex justify-content-between" closeButton={false}>
-                <div className="d-flex align-items-center">
-                    {errIcon}
-                    <strong className="ms-2">{errType}</strong>
-                </div>
-                <button className="toast-close" onClick={handleCloseToast} ref={ref}>
-                    <CgClose className="close-symbol" />
-                </button>
-            </Toast.Header>
-            <Toast.Body>
-                {errMsg}
-            </Toast.Body>
-        </Toast>}
+        {showError
+            && <Toast
+                className="m-3 position-relative err-toast"
+                // makes the toast appear above the grey shadow created by the Modal component
+                style={{zIndex: 9999}}
```

Comment:
I didn't see this issue when testing locally. How would I be able to reproduce this issue? Just to note, the `err-toast` class has `z-index` styling, so it sohulld overlay above the darkened background when `disableControl` is set to `true` for the toast.

Reply:
Oh, I didn't see the `error-toast` stylesheet.

When you add an option and get an error toast with `z-index: 2`, it appears behind the grey shadow as below.
![Screenshot 2023-06-23 at 1 08 04 PM](https://github.com/rikili/ProPlanner/assets/96703115/c7c4e33c-3df0-48f1-97a1-cf9c478ab1b8)

I thought setting `z-index: 9999` seems to make the toast more visible. I didn't know about setting `disableControl` to `true`. I have revised the code using the `disableControl`.

![Screenshot 2023-06-23 at 1 09 21 PM](https://github.com/rikili/ProPlanner/assets/96703115/e9a0511a-8a9e-4c27-95f9-9c698baebc02)